packages/cedar/src/Chart.ts

```diff
@@ -65,6 +70,14 @@ export default class Chart {
     this._data = deepMerge([], data)
   }
 
+  // DOM element
```

Comment:
remove the setters/getters for now. I'm not sure how it would work if someone changed this.